build.sh

```diff
@@ -952,8 +952,11 @@ case "$COMMAND" in
         fi
 
         examples="Simple TableView Migration Backlink GroupedTableView Encryption"
+        versions="0 1 2 3 4 5"
         for example in $examples; do
-            xc "-workspace $workspace -scheme $example -configuration $CONFIGURATION -sdk iphonesimulator build ARCHS=x86_64 ${CODESIGN_PARAMS}"
+            for version in $versions; do
```

Comment:
Should add a check here to ensure `$example` is `"Migration"` so that we don't build the other examples 6 times when it's not necessary. Something like:
```bash
if [ "$example" = "Migration" ];
    # The migration example needs to be built for each schema version
    # to ensure each compiles 
    for version in $versions; do
        xc "-workspace $workspace -scheme $example -configuration $CONFIGURATION -sdk iphonesimulator build ARCHS=x86_64 ${CODESIGN_PARAMS} OTHER_SWIFT_FLAGS='\$(OTHER_SWIFT_FLAGS) -DSCHEMA_VERSION_$version'"
    done
else
    xc "-workspace $workspace -scheme $example -configuration $CONFIGURATION -sdk iphonesimulator build ARCHS=x86_64 ${CODESIGN_PARAMS}"
fi
```